src/Passes/README.md

```diff
@@ -0,0 +1,322 @@
+# Q# Passes for LLVM
+
+This library defines LLVM passes used for analysing, optimising and transforming the IR. The Q# pass library is a dynamic library that can be compiled and ran separately from the
+rest of the project code. While it is not clear whether this possible at the moment, we hope that it will be possible to write passes that enforce the QIR standard.
```

Comment:
Consider making "QIR standard" a link to something like [this](https://github.com/microsoft/qsharp-runtime/tree/main/src/Qir/Runtime) or [this](https://github.com/microsoft/qsharp-language/tree/main/Specifications/QIR).

(I'm not sure that QIR is strong enough to use the word "standard" to it. For me personally the word "standard" is an extremely strong word. Consider replacing with "specification" or similar)

Reply:
Both good suggestions. Done.